students/minsung/users/views.py

```diff
@@ -1,3 +1,46 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
\ No newline at end of file
+from django.http   import JsonResponse
+from django.views  import View
+
+from users.models  import User
+
+email_validator    = re.compile('^[\w]+@[\w]+\.[\w]+$')
+
+password_validator = re.compile('^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{8,}$')
+
+class UserView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+            if email_validator.match(data['email']) == None:
+
+                return JsonResponse({"message": "INVALID EMAIL"}, status=400)
```

Comment:
1. ==None이라는 표현은 잘 사용하지 않습니다. None 자체가 False로 인식되기 때문입니다.
2. 아래 예시가 complie을 사용하지 않았을 때의 예시입니다.
3. if문 다음 공백은 다 제거해 주세요.(if문끼리는 공백 유지)

```suggestion
regex_email = re.match('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$', email)

if not regex_email:              
    return JsonResponse({'message':'Invalid email format'}, status=400)

```